plugins/extractors/metabase/client/client.go

```diff
@@ -39,6 +43,12 @@ func New() Client {
 }
 
 func (c *client) Authenticate(ctx context.Context, host, username, password, sessionID string) error {
+	urlb, err := urlbuilder.NewSource(host)
+	if err != nil {
+		return err
+	}
+
+	c.urlb = urlb
```

Comment:
I think it would be better to keep this in `Init` before calling authenticate.

Reply:
host is also being set here. so in a way, this function is the `Init` for this client.